parsers/verilog.c

```diff
@@ -582,6 +582,7 @@ static void skipToSemiColon (void)
 	} while (c != EOF && c != ';');
 }
 
+/* read an identifier, keyword, number, compiler directive, or macro identifier */
 static bool readIdentifier (tokenInfo *const token, int c)
```

Comment:
How do you think renaming this function to "readToken"?

Reply:
I have been thinking exactly same thing.
I will rename them after this issue is complete.